### PR TITLE
Chef 2424/configure nginx and unicorn

### PR DESCRIPTION
**Pull request by ayush-billore**
### :white_check_mark: Checklist

**All PRs** must tick these:

- [ ] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [ ] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [ ] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [ ] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [ ] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [ ] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [ ] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### CODE REVIEW

File: components/automate-cs-ocid/habitat/default.toml [hunk 1]
```diff
@@ -39,7 +39,7 @@ pool_init_size=5
 pool_queue_max=50
 
 [ocid]
-  [chef_server_config]
+  [ocid.chef_server_config]
```
Comment (RajeshPaul38) [inline components/automate-cs-ocid/habitat/default.toml:42]: This is done in the feature branch for the demo purpose. You can skip this change.

File: components/automate-load-balancer/habitat/config/automate-cs-nginx-location.conf [hunk 2]
```diff
@@ -63,3 +63,19 @@ location ~ ^/(_status|compliance/organizations|organizations|users|authenticate_
   proxy_http_version 1.1;
   proxy_set_header Connection "";
 }
+
+rewrite ^/id$ /id/ permanent;
```
Comment (RajeshPaul38) [inline components/automate-load-balancer/habitat/config/automate-cs-nginx-location.conf:67]: what does it do?

File: components/automate-cs-ocid/habitat/default.toml (hunk 1 above)
Comment (ayush-billore) [reply components/automate-cs-ocid/habitat/default.toml:42]: I will manage it during rebase if required.

File: components/automate-load-balancer/habitat/config/automate-cs-nginx-location.conf (hunk 2 above)
Comment (RajeshPaul38) [reply components/automate-load-balancer/habitat/config/automate-cs-nginx-location.conf:67]: we discussed

**Review by RajeshPaul38 — APPROVED** [review]

**Pull request merged**